managing_providers/_topics/automation_management_providers.md

```diff
@@ -365,3 +365,174 @@ To use the button to run an Ansible Tower job on a virtual machine:
+```
+
+You also need to set the docker image name in advanced settings page before enabling the server role. Navigate to the **Settings** > **Application Settings** in {{ site.data.product.title_short }} UI and set the value for `container_image` field.
+
+An example value of this field is `container_image: docker.io/manageiq/opentofu-runner:latest`.
+
+## Usage of Embedded Terraform (OpenTofu)
+
+The following sections show the usage of Embedded Terraform in {{ site.data.product.title_short }}. The following sections apply to {{ site.data.product.title_short }} that is deployed as a containerized deployment (podified) and {{ site.data.product.title_short }} that is deployed as a virtual machine appliance:
+
+1. Enable the Embedded Terraform server role.
+2. Add a source control repository that contains your templates.
+3. Add credentials for your cloud providers.
+4. Create a Service Catalog item with the wanted Terraform template.
+
+### Enabling the Embedded Terraform Server Role
+
+In {{ site.data.product.title_short }}, the Embedded Terraform server role is disabled by default. Enable this server role to use Terraform Automation Inside.
+
+To enable the Embedded Terraform Server Role, use the following steps:
+
+1. Browse to the settings menu, and click **Configuration** > **Settings**.
+2. Select the wanted server under **Zones**.
+3. Set the **Server Role** for **Embedded Terraform** to `On`.
+
+### Verifying the Embedded Terraform worker state
+
+Verify that the Embedded Terraform worker is started to use its features:
+
+1. Browse to the settings menu, then click **Configuration** > **Diagnostics** and click the server that you want to choose.
+2. Click **Roles by Servers** tab.
+
+A table of all workers and their status appears from which you can confirm the state of your Embedded Terraform worker.
+
+### Adding a Template Repository
+
+To enable {{ site.data.product.title_short }} to discover and use your Terraform templates, add a repository to store and manage your templates. Use the following steps to add a repository:
+
+1. Browse to the menu and click **Automation > Embedded Terraform > Repositories**.
+
+2. Click **Configuration**, then ![Add New Repository](../images/1862.png) (**Add New Repository**).
+
+3. Provide a Repository Name in the **Name** field.
+
+4. Add a description for the repository in the **Description** field.
+
+5. Add a **URL** or IP address for the repository.
+
+6. Select the appropriate **SCM Credentials** from the drop-down menu.
+
+7. Provide a branch name in the **SCM Branch** field.
+
+8. Click **Save**.
+
+When you sync a repository, the Terraform templates become available to {{ site.data.product.title_short }}.
+
+### Refreshing Repositories
+
+You can use {{ site.data.product.title_short }} to refresh targeted Terraform templates or all repositories in your inventory to make sure that your templates are up to date.
```

Comment:
I think it should be "refresh a specific Terraform repository" not "templates".  We can't sync individual templates and the alternative here talks about repositories anyway.

```suggestion
You can use {{ site.data.product.title_short }} to refresh specific Terraform repositories or all repositories in your inventory to make sure that your templates are up to date.
```